docs/contributors/maintainers.md

```diff
@@ -59,17 +72,11 @@ A GitHub job is run at git tag creation, so the publishing should be done automa
   of the `Done` column related to the milestone
 
 
-### Prepare the GitHub Release
+### Continue filling the GitHub Release
 
 - Open [github releases](https://github.com/process-analytics/bpmn-visualization-js/releases)
-- Create a new draft release and name it `Next` (the name is not relevant and will be replaced automatically later).
-This ensures that development can continue without impacting the writing of the content of the in progress release. That way,
-if a PR is merged, `release-drafter` will update the `Next` draft release keeping the in-progress release untouched.
-- The draft release for the newly tagged version should already exist:
-  - [release-drafter](https://github.com/release-drafter/release-drafter) creates or updates draft release for the
-  next version each time a pull request is merged to the `master` branch.
-  - create a new release if it is missing or rename the existing one to match .
-- Assign the new tag as release target and save the draft (this should have already been managed by `release-drafter`)
+- Rename the existing `In Progress` draft release (updated before starting the release) to match the version that has just been tagged
```

Comment:
I feel that we are repeating ourselves from the previous section.

Reply:
Previously, we said to update the name to `In Progress`, now we say to update it to he actual version.
We do the rename twice: the goal is to detect if release-drafter is updating the next release and not the in progress one.